src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/DeliverySessionController.kt

```diff
@@ -95,6 +97,7 @@ class DeliverySessionController(
     return DeliverySessionDTO.from(deliverySession)
   }
 
+  @Deprecated("superseded by PUT /referral/{referralId}/delivery-session-appointments/{appointmentId}/attendance")
```

Comment:
Added ticket to track these https://trello.com/c/UqV3VQGn/192-remove-deprecated-endpoints